src/main/java/uk/ac/ebi/atlas/experiments/ExperimentsListController.java

```diff
@@ -16,8 +19,9 @@ public ExperimentsListController(ExperimentInfoListService experimentInfoListSer
 
     //Used by experiments table page
     @GetMapping(value = "/json/experiments",
-                produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
-    public String getExperimentsList() {
-        return GSON.toJson(experimentInfoListService.getExperimentsJson());
+            produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+    public String getExperimentsList(@RequestParam(defaultValue = "") String organism_part) {
+        return isNullOrEmpty(organism_part) ? GSON.toJson(experimentInfoListService.getExperimentsJson()) :
```

Comment:
Do you need to check if `organismPart` is empty? What would happen if you pass an empty string?